scripts/mergeModels.sh

```diff
@@ -24,8 +24,8 @@ MODELS="Models.swift"
 mergeFiles() {
     TEMP=`mktemp`
     cat *.swift | grep -e "//\ Code\ generated" | sort | uniq >> ${TEMP}
-    cat *.swift | grep -e "^import\ " | sort | uniq  >> ${TEMP}
-    cat *.swift | grep -ve "^import\ " | grep -ve "//\ Code\ generated" >> ${TEMP}
+    cat *.swift | grep -e "^import\ " -e "@_spi" | sort | uniq  >> ${TEMP}
+    cat *.swift | grep -ve "^import\ " -ve "@_spi" -ve "//\ Code\ generated" >> ${TEMP}
```

Comment:
Without this change, in the merged model file result, there would be dozens of `@_spi` statements. This ensures there's no no duplicate SPI imports in merged file.

Reply:
This is great.  When I implemented `@_spi()` in codegen a year ago, I kicked the can on this.  Thanks for mopping it up!